Source/System/Drawing/ColorConverter.cs

```diff
@@ -0,0 +1,433 @@
+                            else if ((text.Length == 7 && text[0] == '#') ||
+                                     (text.Length == 8 && (text.StartsWith("0x") || text.StartsWith("0X"))) ||
+                                     (text.Length == 8 && (text.StartsWith("&h") || text.StartsWith("&H")))) {
+                                // Note: ConvertFromString will raise exception if value cannot be converted.
+                                obj = Color.FromArgb(unchecked((int)(0xFF000000 | (uint)(int)intConverter.ConvertFromString(context, culture, text))));
+                            }
+                        }
+
+                        // Nope.  Parse the RGBA from the text.
+                        //
+                        if (obj == null) {
+                            string[] tokens = text.Split(new char[] {sep});
+                            int[] values = new int[tokens.Length];
+                            for (int i = 0; i < values.Length; i++) {
+                                values[i] = unchecked((int)intConverter.ConvertFromString(context, culture, tokens[i]));
+                            }
+
+                            // We should now have a number of parsed integer values.
+                            // We support 1, 3, or 4 arguments:
+                            //
+                            // 1 -- full ARGB encoded
+                            // 3 -- RGB
+                            // 4 -- ARGB
+                            //
+                            switch (values.Length) {
+                                case 1:
+                                    obj = Color.FromArgb(values[0]);
+                                    break;
+
+                                case 3:
+                                    obj = Color.FromArgb(values[0], values[1], values[2]);
+                                    break;
+
+                                case 4:
+                                    obj = Color.FromArgb(values[0], values[1], values[2], values[3]);
+                                    break;
+                            }
+                            tryMappingToKnownColor = true;
+                        }
+
+                        if ((obj != null) && tryMappingToKnownColor) {
+
+                            // Now check to see if this color matches one of our known colors.
+                            // If it does, then substitute it.  We can only do this for "Colors"
+                            // because system colors morph with user settings.
+                            //
+                            int targetARGB = ((Color)obj).ToArgb();
+
+                            foreach (Color c in Colors.Values) {
+                                if (c.ToArgb() == targetARGB) {
+                                    obj = c;
+                                    break;
+                                }
+                            }
+                        }
+                    }
+#if NETSTANDARD20
```

Comment:
I have one question.

Whole file seems to be surrounded by `NETSTANDARD20`.
Is this `#if` - `#endif` necessary ?

Reply:
We could solve this bt excluding this file in csproj for other targets.This file is only needed for netstandatd2.0 target frameworks, other targets defined in csproj has this class available in corefx framework. It is important to note that this is not only about supporting netstandard2.0 but also supporting target frameworks that implement the nonstandard2.0 but are not defined in csproj target frameworks, this includes things like android and iOS targets.

Reply:
@wieslawsoltes 

Thank you for your response.
But I don't know if this is necessary...

ColorConverter.cs
```cs
// https://github.com/dotnet/runtime/blob/master/src/libraries/System.Drawing.Common/src/System/Drawing/ColorConverter.cs
#if NETSTANDARD20
// Licensed to the .NET Foundation under one or more agreements.
...
#if NETSTANDARD20
...
#else
...
#endif
...
#endif
```

If `NETSTANDARD20` is undefined, it looks like entire file is disabled.

Reply:
I prefer if/defs in .cs file than messing with csproj and conditional compilation.

Reply:
I think the question was about the nested ifdefs, which you has answered [above](https://github.com/vvvv/SVG/pull/630#discussion_r362266539).

Reply:
@mrbean-bremen 

Thanks !

@wieslawsoltes 

I understood.
You changed `ColorConverter.cs` from `dotnet/runtime` to solve this issue.
This if-else-endif was confusing to me...
It's not original file, so please put a hint at top of file.

For example:
```cs
// This file is based on
// https://github.com/dotnet/runtime/blob/master/src/libraries/System.Drawing.Common/src/System/Drawing/ColorConverter.cs
// 6a988c7d0389bf9e9aa20d23baa353e9393b2ea5
```

Reply:
I will submit updated comment as proposed.